src/components/MovieList/MovieList.jsx

```diff
@@ -1,34 +1,49 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
 import Axios from 'axios';
+import classNames from 'classnames';
 import MovieItem from '../MovieItem/MovieItem';
 import styles from './movie-list.scss';
 
+const cx = classNames.bind(styles);
+
 class MovieList extends Component {
+  static getMovies() {
+    return Axios.get('http://localhost:3000/api/v1/movies');
```

Comment:
Let's create separate file api,js wich receives data, makes request and returns response. Also as we use babel env preset and es2017 contains async await spec I would like to use it